pytorch_lightning/trainer/trainer.py

```diff
@@ -1113,25 +1124,19 @@ def _run(
              Lightning internal flow looks like this:
         {Trainer.fit} or {Trainer.test} or {Trainer.predict}  ||
                                 |                             ||
-                        create accelerator                    ||
-                                |                             ||
-                         {self._dispatch}                     ||
+                        setup accelerator                     ||
+                           and strategy                       ||
```

Comment:
should we mention that this may include spawning processes?

Reply:
Yes, I would prefer so.

Reply:
updated with your suggestions